### ISSUE TITLE
Utiliser le domaine pour la liste de diffustion

### ISSUE THREAD

**Issue by MicroJoe** [haum/haum_internal#33]
Actuellement la liste est hébergée sur le domaine matael.org alors qu'il serait beaucoup plus joli d'avoir un truc du genre <lists.haum.org> (voire même <listes.haum.org> en mode francophone).

Je vois dans ma boule de cristal une entrée DNS à rajouter et à la gérer sur le serveur cible, que ce soit celui de matael ou alors un dédié du HAUM (mais je ne sais pas si ils ont étés réinstallés).

**Comment by jblb** [comment]
je pense que ta boule de ristal ne vois pas trop mal, je me demande juste pourquoi on ne l'a pas fait avant.

**Comment by fredblain** [comment]
On en a déjà parlé mais il n'y avait pas eu de suite. Probablement parce qu'on avait décidé de faire ça bien en commençant par arrêter d'utiliser le serveur de @Matael  ;)

**Comment by MicroJoe** [comment]
Mais l'utilisation d'un autre serveur n'est pas nécessaire pour effectuer ce changement de domaine du point de vue technique ; je pense donc que l'on peut s'en occuper sans attendre une migration comlpète des serveurs.

Je vais voir pour créer une autre issue pour la migration des serveurs (j'ai déjà fait de l'admin sys, je peux aider éventuellement).

**Comment by fredblain** [comment]
Je suis d'accord avec toi, mais à l'époque, l'idée était de tout faire ensemble, proprement. Bon après, il s'avère que ça ne s'est pas fait. J'ai simplement voulu dire qu'on en avait déjà parlé :)

Bonne initiative et ton aide sera assurément appréciée bien que d'autres maîtrisent également l'admin-sys. LE pb étant qu'on a pas le serveur sur lequel faire tourner tout ça (mailing-lists, site web, etc.). Ou alors il est en panne. Soit on en loue un avec un coût certain sur le budget du HAUM, soit en trouve un/on répare celui qu'on a (pb d'alim il me semble).

**Comment by neomilium** [comment]
Non, le serveur est en place, c'est le secondaire qui est en panne. Par
contre, il faut finaliser la configuration et installer le site dessus..

**Comment by fredblain** [comment]
OK, au temps pour moi. Merci @neomilium pour la précision.

**Comment by jblb** [comment]
le serveur est bien en place mais pour l'instant les redirection de ports extérieurs ne sont pas mise en place.  Donc ce brave serveur n'est accessible que depuis le réseau qui est au local.....

**Comment by MicroJoe** [comment]
Ce qui ne nous empêche pas de faire une redirection DNS en attendant car ça sera vraiment plus rapide que de travailler sur le serveur, autant faire une seule chose à la fois. Je peux créer une autre issue pour la mise en place du serveur pour bien distinguer si vous voulez.

**Comment by MicroJoe** [comment]
Après discussion, il faut en plus de la redirection DNS reconfigurer Mailman afin de prendre en compte le nouveau NDD. Ça peut paraitre pénible (quoique...) mais ça permettrai d'avoir une liste propre à l'assoce.

**Comment by neomilium** [comment]
+1

Le 20 novembre 2015 08:54, Romain Porte <notifications@github.com> a écrit :

> Après discussion, il faut en plus de la redirection DNS reconfigurer
> Mailman afin de prendre en compte le nouveau NDD. Ça peut paraitre pénible
> (quoique...) mais ça permettrai d'avoir une liste propre à l'assoce.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/haum/haum_internal/issues/33#issuecomment-158314977>.
>



-- 
Romuald

**Comment by jblb** [comment]
-----BEGIN PGP SIGNED MESSAGE-----
Hash: SHA512

Je propose de le faire dans un premier temps sur la liste du bureau pour évaluer les éventuelles difficulté à faire le changement

Le 20 novembre 2015 09:40:12 CET, Romuald Conty <notifications@github.com> a écrit :
>+1
>
>Le 20 novembre 2015 08:54, Romain Porte <notifications@github.com> a
>écrit :
>
>> Après discussion, il faut en plus de la redirection DNS reconfigurer
>> Mailman afin de prendre en compte le nouveau NDD. Ça peut paraitre
>pénible
>> (quoique...) mais ça permettrai d'avoir une liste propre à l'assoce.
>>
>> —
>> Reply to this email directly or view it on GitHub
>>
><https://github.com/haum/haum_internal/issues/33#issuecomment-158314977>.
>>
>
>
>
>--
>Romuald
>
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/haum/haum_internal/issues/33#issuecomment-158325712

- --
Envoyé de mon appareil Android avec K-9 Mail. Veuillez excuser ma brièveté.
-----BEGIN PGP SIGNATURE-----
Version: APG v1.1.1

iQFABAEBCgAqBQJWTt6oIxxKZXJvbWUgQi4gKEpiTGIpIDxqZXJvbWVAamJsYi5u
ZXQ+AAoJELfBan+yggskPqsIAKpgt8xClIl4Iq1Kz3FXL7exE50SnhcF1tKLY2l3
IQjZM2EWcLahBP6XqoveXnTgcR30E+kqTRpE2oOkZiZuG7n6jhwZh3AyIfNCc1vj
5FNGXXbA8m/Bvrdbl1khdeW0lF3YiiCQYnb53zjnHAOkD6dJMzB+ndcUbnbg7Gvh
ja5DTy9uuDzikmZ3a51gKusWyQSXKdaKrglTG+kU3ckV7v4FoyMMfDAEUHtzpBA6
pf076HiekhqzlBvsSTY6ydUMOtIpujqDUkm1oyu/JT2Q8A7jQMgbs0zXl7GJXW+8
ax/ztdgY7nafnYf1UqWjm0K7cFwSb8TcboIa0gp7+vLrcoA=
=JvJA
-----END PGP SIGNATURE-----

**Comment by neomilium** [comment]
Bonne idée.
N'hésitez pas à décrire le processus sur le wiki :)

Le 20 novembre 2015 09:49, JbLb <notifications@github.com> a écrit :

> -----BEGIN PGP SIGNED MESSAGE-----
> Hash: SHA512
>
> Je propose de le faire dans un premier temps sur la liste du bureau pour
> évaluer les éventuelles difficulté à faire le changement
>
> Le 20 novembre 2015 09:40:12 CET, Romuald Conty <notifications@github.com>
> a écrit :
> >+1
> >
> >Le 20 novembre 2015 08:54, Romain Porte <notifications@github.com> a
> >écrit :
> >
> >> Après discussion, il faut en plus de la redirection DNS reconfigurer
> >> Mailman afin de prendre en compte le nouveau NDD. Ça peut paraitre
> >pénible
> >> (quoique...) mais ça permettrai d'avoir une liste propre à l'assoce.
> >>
> >> —
> >> Reply to this email directly or view it on GitHub
> >>
> ><https://github.com/haum/haum_internal/issues/33#issuecomment-158314977>.
> >>
> >
> >
> >
> >--
> >Romuald
> >
> >
> >---
> >Reply to this email directly or view it on GitHub:
> >https://github.com/haum/haum_internal/issues/33#issuecomment-158325712
>
> - --
> Envoyé de mon appareil Android avec K-9 Mail. Veuillez excuser ma brièveté.
> -----BEGIN PGP SIGNATURE-----
> Version: APG v1.1.1
>
> iQFABAEBCgAqBQJWTt6oIxxKZXJvbWUgQi4gKEpiTGIpIDxqZXJvbWVAamJsYi5u
> ZXQ+AAoJELfBan+yggskPqsIAKpgt8xClIl4Iq1Kz3FXL7exE50SnhcF1tKLY2l3
> IQjZM2EWcLahBP6XqoveXnTgcR30E+kqTRpE2oOkZiZuG7n6jhwZh3AyIfNCc1vj
> 5FNGXXbA8m/Bvrdbl1khdeW0lF3YiiCQYnb53zjnHAOkD6dJMzB+ndcUbnbg7Gvh
> ja5DTy9uuDzikmZ3a51gKusWyQSXKdaKrglTG+kU3ckV7v4FoyMMfDAEUHtzpBA6
> pf076HiekhqzlBvsSTY6ydUMOtIpujqDUkm1oyu/JT2Q8A7jQMgbs0zXl7GJXW+8
> ax/ztdgY7nafnYf1UqWjm0K7cFwSb8TcboIa0gp7+vLrcoA=
> =JvJA
> -----END PGP SIGNATURE-----
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/haum/haum_internal/issues/33#issuecomment-158327771>.
>



-- 
Romuald
